### PR TITLE
gdu: rename binary to gdu-go to mitigate conflict with coreutils

### PR DESCRIPTION
**Pull request by dundee**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/gdu.rb [hunk 1]
```diff
@@ -17,8 +17,6 @@ class Gdu < Formula
 
   depends_on "go" => :build
 
-  conflicts_with "coreutils", because: "both install `gdu` binaries"
-
```
Comment (carlocab) [inline Formula/gdu.rb:21]: Let's remove this in a separate PR. Removing it here means we'll also need to remove it from `coreutils` but that just results in an unnecessary rebuild.

Comment (dundee) [reply Formula/gdu.rb]: Ah, I see. I will create second PR for that.

**Review by carlocab — COMMENTED** [review]
Thanks, will be merged in #134722.

**Comment by carlocab** [comment]
The other PR should merge soon. When you open your next PR, please ask for the `CI-syntax-only` label to be applied. Thanks!

**Comment by dundee** [comment]
Thanks! Next PR in https://github.com/Homebrew/homebrew-core/pull/134731

**Pull request closed without merging**